fv3core/pace/fv3core/stencils/d_sw.py

```diff
@@ -921,6 +927,12 @@ def make_quantity():
                 },
             )
         )
+        self._accumulate_heat_source_and_dissipation_estimate_stencil = (
```

Comment:
what's the reason for splitting this into a separate stencil?

Reply:
Readability mostly I believe. Also potentially deactivating it entirely - which we didn't setup

Reply:
It looks like heat_source and diss_est are reset to 0 in the original implementation, but that doesn't look to be happening here. Also do the tmp_diss_est and tmp_heat_source variables need to be reset to 0 between calls as well? I'm also not sure we want to do this as a separate stencil --can we put a compile-time if to deactivate this if need be?

Reply:
We can but it wasn't done in the original port because... it was not ported at all right? We excluded the heat accumulation from the translate test because we didn't port the code doing the work. In my book this an improvement on the port, there's no reason to have an option to do that, or am I missing something?